src/Client/Shared/ApiHelper.cs

```diff
@@ -38,6 +39,13 @@ public static class ApiHelper
         {
             snackbar.Add(ex.Result.Exception, Severity.Error);
         }
+        catch (ApiException ex)
```

Comment:
Where/when exactly do you get into this catch block without it being catched by the ApiException<ErrorResult> block before? The api should always send an errorresult... Or maybe it's in the case of not even getting to the api? Still... when does this happen?

Reply:
must i close this issue/pr?

Reply:
Not necessarily... We should find the actual reason why you're having this problem...

How exactly do you repeat this? Tell the exact steps you take.

Reply:
this front-end (current version) cannot deserialize `ApiException<ErrorResult>` lost so catch

Reply:
Yes but *why* can it not deserialize it? And *when* exactly does this happen? **Which exact steps did you take to get here?**

Reply:
for example 
![image](https://user-images.githubusercontent.com/27079402/149645011-5e31b47d-eb61-49af-8357-2c9a2a0435a0.png)

Reply:
![image](https://user-images.githubusercontent.com/27079402/149645066-db6c17ab-d9bd-4ca3-8be4-f2eef7689d3b.png)

Reply:
when you send a wrong field then api responde with status different to 200 with exception message